examples/supplychain-app/quorum/express_nodeJS/controllers/container.js

```diff
@@ -15,6 +16,7 @@ router.get("/:trackingID?", function(req, res) {
     // TODO: Implement getContainerByID functionality
     const trackingID = req.params.trackingID;
     console.log(trackingID, "***");
+    console.log(Protocol,"*******");
```

Comment:
is the `console.log` required?

Reply:
Would be good to have it.. can remove incase harmful